components/connector-service/internal/externalapi/revocationhandler.go

```diff
@@ -0,0 +1,68 @@
+package externalapi
+
+import (
+	"github.com/kyma-project/kyma/components/connector-service/internal/apperrors"
+	"github.com/kyma-project/kyma/components/connector-service/internal/certificates"
+	"github.com/kyma-project/kyma/components/connector-service/internal/certificates/revocationlist"
+	"github.com/kyma-project/kyma/components/connector-service/internal/httphelpers"
+	"net/http"
+)
+
+const (
+	CertificateHeader = "Certificate-Header"
```

Comment:
I suggest changing the header name to something like `Client-Certificate`

Reply:
Done